### ISSUE TITLE
Accidental duplication of removed icons in last major release

### ISSUE THREAD

**Issue by mondeja** [simple-icons/release-action#25]
This happened at the launch of v4.0.0, as shows the original comment in the [pull request simple-icons/simple-icons/4150](https://github.com/simple-icons/simple-icons/pull/4150). Removed icons also were counted twice in the title.

**Comment by mondeja** [comment]
It seems that the problem is check the data file for removals. If I'm not wrong, the next release will be marked as breaking change because there are two icons that have been renamed in pull requests. This is the current return of `removedIcons` in `filterDuplicates` function:

```javascript
[
  {
    id: '8Kahoot',
    changeType: 'removed',
    name: 'Kahoot',
    prNumbers: [ 4635 ]
  },
  {
    id: '118Big Basket',
    changeType: 'removed',
    name: 'Big Basket',
    prNumbers: [ 4616 ]
  }
]
```

So my purposed change is: do not trust in data file for removals. Maybe the removal of [these lines](https://github.com/simple-icons/release-action/blob/31327bdb2fad8dee207d5682e526bf79fd00f49c/src/create.js#L339-L348) is the solution. Do you trust in data file to count removals for some reason @ericcornelissen?
-

**Comment by ericcornelissen** [comment]
> It seems that the problem is to check the data file for removals. If I'm not wrong, the next release will be marked as breaking change because there are two icons that have been renamed in pull requests.

Yep, that is [exactly what I thought](https://github.com/simple-icons/simple-icons/pull/3611#issuecomment-755273090) for Kahoot. Wasn't aware of #4616 though 🤔 For more [see this comment](https://github.com/simple-icons/simple-icons/pull/4616#issuecomment-757068815).

> So my purposed change is: do not trust in data file for removals counting. Maybe the deletion of [this block](https://github.com/simple-icons/release-action/blob/31327bdb2fad8dee207d5682e526bf79fd00f49c/src/create.js#L339-L348) is the solution. Do you trust in data file to count removals for some reason @ericcornelissen?

That seems like a reasonable suggestion and may also fix this issue.

> Also, is the renaming of icons a breaking change? I'm not even sure 😅

In most cases it is, but indeed not always (as is the case with Kahoot). Either because the filename changes or because the `simpleIcons.get` functionality no longer works for the old name (though this may change, at least to an extend, with https://github.com/simple-icons/simple-icons/issues/2260).

**Comment by ericcornelissen** [comment]
A little bit of a delay but I was finally able to look into this and it should be now fixed with 40976a5f554c9c4f4c6f7e1cfc83a04b94595fa8 thanks to [this suggestion](https://github.com/simple-icons/release-action/issues/25#issuecomment-757061527) by @mondeja :tada: 

Also uncovered some "faulty" tests (more accurately, the mock for the GitHub API didn't quite work as the real GitHub API does). After fixing that I saw the duplicate in the tests as well and after implementing @mondeja's suggestion the duplicates disappeared.

**Comment by mondeja** [comment]
Thank you for the great work @ericcornelissen!
